packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -8,17 +8,17 @@ import {
   Prop
 } from '@stencil/core';
 
-/**
- * This is a wrapper component whose job is to marshal the cells in slotted va-table-rows 
- * through DOM manipulation for projection into the slots of the va-table-inner component for rendering
- */
 
 @Component({
   tag: 'va-table',
   styleUrl: 'va-table.scss',
   shadow: true,
 })
 export class VaTable {
+  /**
+  * This is a wrapper component whose job is to marshal the cells in slotted va-table-rows 
+  * through DOM manipulation for projection into the slots of the va-table-inner component for rendering
+  */ 
```

Comment:
This comment block was displaying on the Storybook page itself so I'm moving it into the class to stop that from happening.

![Screenshot 2024-06-11 at 10 27 23 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/cec2a831-89fd-4676-a469-5a3f8f40f51b)
![Screenshot 2024-06-11 at 10 27 23 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/cec2a831-89fd-4676-a469-5a3f8f40f51b)